content/operate/oss_and_stack/management/security/acl.md

```diff
@@ -274,17 +274,7 @@ really annoying, so instead we do things like this:
 
     > ACL SETUSER antirez on +@all -@dangerous >42a979... ~*
 
-By saying +@all and -@dangerous, we included all the commands and later removed
-all the commands that are tagged as dangerous inside the Redis command table.
-Note that command categories **never include modules commands** with
-the exception of +@all. If you say +@all, all the commands can be executed by
```

Comment:
Please mention that @all includes all module commands because this is true also for 3rd party modules users may choose to add to their Redis CE server.

Reply:
Just that one line or the entire paragraph?

Reply:
all 277-280

Reply:
Fixed in next push.